src/styles/headerCell.css

```diff
@@ -0,0 +1,25 @@
+.cell {
+    composes: cell from './shared/cell.css';
+}
+
+.sortable {
+    cursor: pointer;
+}
+
+.sortArrow {
+    width: 16px;
+    height: 16px;
+    background-image: url('images/ui-icons_222222_256x240.png');
```

Comment:
I don't think we should be adding this png sprite file. We should use unicode as @rishson suggested until we've created an icon widget / chosen an icon approach for widgets